api/resources/translations/messages-es.properties

```
@@ -325,9 +325,9 @@ analytics.reporting.date_range = Rango de la fecha
 analytics.reporting.district = Escoger distrito
 analytics.reporting.form = Escoger formulario
 analytics.reporting.health_centers_size = SMO
-analytics.reporting.incomplete = Informes invalidos
+analytics.reporting.incomplete = Informes inválidos
 analytics.reporting.last_time_unit = Últimos {{quantity}} {{time_unit}}
-analytics.reporting.not_submitted = Informes que faltan
+analytics.reporting.not_submitted = Informes que cargados
```

```suggestion
analytics.reporting.not_submitted = Informes faltantes
```
The intention is missing reports (not_submitted)